File: src/test/java/br/com/jiratorio/config/FakerConfig.java
package br.com.jiratorio.config;

import com.github.javafaker.Faker;
import java.util.Random;
import lombok.extern.slf4j.Slf4j;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Slf4j
@Configuration
public class FakerConfig {

    @Bean
    public Faker faker() {
        long seed = System.currentTimeMillis();
        log.info("I=generated seed, seed={}", seed);
        return new Faker(new Random(seed));
    }

}

File: src/test/kotlin/br/com/jiratorio/config/FakerConfig.kt
package br.com.jiratorio.config

import br.com.jiratorio.extension.log
import com.github.javafaker.Faker
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import java.util.Random

@Configuration
class FakerConfig {

    @Bean
    fun faker() = Faker(Random(System.currentTimeMillis().also {
        log.info("I=generated seed, seed={}", it)
    }))

}
